pkgs/applications/networking/instant-messengers/cordless/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "cordless";
+  version = "2020-08-30";
+
+  src = fetchFromGitHub {
+    owner = "Bios-Marcel";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-CwOI7Ah4+sxD9We+Va5a6jYat5mjOeBk2EsOfwskz6k=";
+  };
+
+  subPackages = [ "." ];
+
```

Comment:
default
```suggestion
```

Reply:
It's not the default.

With:
```
$  lsta ./result/
./result
└── bin
   └── cordless
```
Without:
```

$  lsta ./result/
./result
└── bin
   ├── box
   ├── button
   ├── checkbox
   ├── cordless
   ├── dropdown
   ├── flex
   ├── form
   ├── frame
   ├── grid
   ├── inputfield
   ├── inputfield-paste
   ├── keyevents
   ├── layout
   ├── list
   ├── modal
   ├── overflowindicators
   ├── pages
   ├── presentation
   ├── primitive
   ├── table
   ├── textview
   ├── theme
   ├── treeview
   └── unicode
```

Reply:
I take that back then